core/src/main/java/com/tweet/core/model/User.java

```diff
@@ -13,6 +12,10 @@
     private String firstName;
     private String surname;
 
+    @Enumerated(EnumType.STRING)
```

Comment:
UserRole is a POJO entity not enum type, but having enum in UserRole as name (or role) would be nice.
See comment for UserRole class.